anywidget/widget.py

```diff
@@ -3,6 +3,21 @@
 
 from ._version import __version__
 
+DEFAULT_ESM = """
+    export function render(view) {
+        console.log("Dev note: No _esm defined for this widget:", view);
+        let root = document.createElement("div");
+        let url = "https://anywidget.dev/en/getting-started/";
+        let p = Object.assign(document.createElement("p"), {
+            innerHTML: '<strong>Dev note</strong>: ' +
+            `<a href='${url}' target='blank'>Implement an <code>_esm</code> attribute</a>` +
+            ` on AnyWidget subclass <code>${view.model.get('_anywidget_id')}</code>` +
+            ' to customize this widget.'
+        });
+        root.appendChild(p);
+        view.el.appendChild(root);
+    }
```

Comment:
Slight nit, we don't really need to create these elements with. I think this renders correctly if you just assign the `view.el.innerHTML` directly (the back-ticks string syntax supports multi-lines):

```suggestion
DEFAULT_ESM = """
export function render(view) {
  console.log("Dev note: No _esm defined for this widget:", view);
  let url = "https://anywidget.dev/en/getting-started/";
  view.el.innerHTML = `<p>
    <strong>Dev note</strong>:
    <a href='${url}' target='blank'>Implement an <code>_esm</code> attribute</a>
    on AnyWidget subclass <code>${view.model.get('_anywidget_id')}</code>
    to customize this widget.
  </p>`;
}
"""

Reply:
<img  alt="image" src="https://user-images.githubusercontent.com/24403730/216400788-3cab4f42-da2b-4503-a34b-fe36a9052e4b.png">